docs/deployment-examples.md

```diff
@@ -0,0 +1,331 @@
+    $ cat /etc/systemd/system/purefa-ome.service
+    ```
+
+    Example:
+    ```console
+    [Unit]
+    Description=Pure Storage FlashArray OpenMetrics Exporter
+    After=network.target
+    StartLimitIntervalSec=0
+    [Service]
+    Type=simple
+    Restart=always
+    RestartSec=1
+
+    # Start OME
+    ExecStart=/usr/bin/pure-fa-om-exporter --port 9490
+
+    # Start OME with TLS
+    # ExecStart=/usr/bin/pure-fa-om-exporter --port 9490 -c /etc/pki/tls/certs/purefa-ome/pure-ome.crt -k /etc/pki/tls/private/pure-ome.key
+
+    [Install]
+    WantedBy=multi-user.target 
+    ```
+
+8. **Start, enable and test the service**
+    Reload the system daemon to read in changes to services
+
+    ```console
+    $ systemctl daemon-reload
+    ```
+
+    Start the service and check the status is successful
+    
+    ```console
+    $ systemctl start purefa-ome
+    $ systemctl status purefa-ome
+    ```
+    
+    Enable the service to start on OS startup
+    ```console
+    $ systemctl enable purefa-ome
+    ```
+
+## Binary - Tokens File - http with API tokens file embedded in exporter
+Similar steps as basic but we just need to cover a couple of minor changes to running and testing the deployment
+Follow steps 1-4 and 7-8 of the default binary deployment, but substitute the following steps for executing and testing.
+
+4. **Create a tokens.yaml file to pass to the exporter**
+
+    ```console
+    $ cat /directorypath/tokens.yaml
+    # alias: of the array to be used in the exporter query
+    array01:
+      # FQDN or IP address of the array
+      address: 'array01.fqdn.com'
+      # API token of a user on the array
+      api_token: 'a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f6'
+
+    # Example of a second array
+    array02:
+      address: 'array02.fqdn.com'
+      api_token: 'a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f7'
+    ```
+
+5. **Run the binary**
+    ```console
+    $ ls out/bin
+    $ .out/bin/pure-fa-openmetrics-exporter --tokens /directorypath/tokens.yaml
+    Start Pure FlashArray exporter v1.0.9 on 0.0.0.0:9490
+    ```
+
+6. **Test the exporter**
+    Use `curl` to test the exporter returns results.
+    ```console
+    $ curl -X GET 'http://localhost:9490/metrics/array?endpoint=gse-array01' -silent | grep ^purefa_info
+    purefa_info{array_name="ARRAY01",os="Purity//FA",system_id="12345678-abcd-abcd-abcd-abcdef123456",version="6.5.0"} 1
+    ```
+
+## Binary - TLS - https passing API token with query
+Similar steps as basic but we just need to cover a couple of minor changes to running and testing the deployment
+Follow steps 1-4 and 7-8 of the default binary deployment, but substitute the following steps for executing and testing.
+
+Create the certificate and key and pass the exporter the files. There are many different methods of generating certificates which we won't discuss here as each organizations has different standards and requirements.
+
+5. **Pass the certificate and private key to the exporter**
+```console
```

Comment:
Formatting looks wrong here. Indent the console code and ensure there is a blank line after the closing ```

Reply:
Resolved in commit 4010cfcac1c820121ac3f1816ef1ed4180d0595b